denguefever_tw/static/dengue_linebot/config/FSM.json

```diff
@@ -82,7 +82,16 @@
         "unrecognized_msg",
         "user_register_location",
         "wait_register_location",
-        "receive_register_location"
+        "receive_register_location",
+        {
+            "name": "wait_bind_zapper",
+            "on_enter": {
+                "name": "on_enter_wait_bind_zapper",
```

Comment:
You haven't implemented these methods, am I correct?

Reply:
Finished in next commit

Reply:
Ok